otx/algorithms/action/adapters/mmaction/models/backbones/movinet.py

```diff
@@ -0,0 +1,1102 @@
+"""
+Code inspired by:
+https://pytorch.org/vision/stable/_modules/torchvision/models/mobilenetv2.html
+https://pytorch.org/vision/stable/_modules/torchvision/models/mobilenetv3.html
+"""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from collections import OrderedDict
+import torch
+from torch.nn.modules.utils import _triple, _pair
+import torch.nn.functional as F
+from typing import Any, Callable, Optional, Tuple, Union
+from einops import rearrange
+from torch import nn, Tensor
+from yacs.config import CfgNode
+from mmaction.models.builder import BACKBONES
+
+
+class Hardsigmoid(nn.Module):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def forward(self, x: Tensor) -> Tensor:
+        x = (0.2 * x + 0.5).clamp(min=0.0, max=1.0)
+        return x
+
+
+class Swish(nn.Module):
```

Comment:
Why don't we use torch.nn.SiLU

Reply:
Replaced with nn.SiLU and also HardSigmoid is replaced with nn.HardSigmoid. Thank you.